code/__HELPERS/game.dm

```diff
@@ -425,7 +425,7 @@
 	var/list/result = list()
 	for(var/m in group)
 		var/mob/M = m
-		if(!M.key || !M.client || (ignore_category && GLOB.poll_ignore[ignore_category] && M.ckey in GLOB.poll_ignore[ignore_category]))
+		if(!M.key || !M.client || (ignore_category && GLOB.poll_ignore[ignore_category] && (M.ckey in GLOB.poll_ignore[ignore_category])))

```

Comment:
What does this fix?

Reply:
`x && y in z` doesn't work how you expect it to, dreamchecker will warn about it and this change disambiguates the operations being done.